JobOfferService/JobOfferService.Model/Profile.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace JobOfferService.Model;
+
+public class Profile : Document
```

Comment:
Profile ima id pod kojim je zapisan u Mongo bazi, pretpostavljam da ce svoje info preuzimati iz ProfileService, gde nam je zapisan u Postgres, nisam sigurna da li mu onda mozda treba dodatno i taj id posto je tamo guid (mada vrlo moguce da moze da se konvertuje u objectId)

Reply:
U pravu si. Dodacu to kad budem radio sync sa drugim servisom. Nesto kao GlobalId